dist/docs_data.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, doxygen_name, data_structures, files):
     ArchDocPage('arch-checkpoint',
         ['WT_CONNECTION'],
         ['src/block/block_ckpt.c', 'src/block/block_ckpt_scan.c',
-         'src/conn/conn_ckpt.c', 'src/meta/meta_ckpt.c',
-         'src/txn/txn_ckpt.c']),
+         'src/checkpoint/checkpoint_conn.c', 'src/meta/meta_ckpt.c',
+         'src/checkpoint/checkpoint_txn.c']),
```

Comment:
These should be re-ordered so that the two in `src/checkpoint` are together and `/src/meta` comes after.

Reply:
Thanks, done in 7c852c4eae0113d7c6b2365a8606937a80612c32.